utils/slices.go

```diff
@@ -156,3 +156,22 @@ func MountSliceContains(s []specs.Mount, x specs.Mount, match func(a, b specs.Mo
 	}
 	return false
 }
+
+// DeviceSliceRemove removes from slice 's' any elements which occur on slice 'db'; the
+// given function is used to compare elements.
+func DeviceSliceRemove(s, db []specs.LinuxDevice, cmp func(d1, d2 specs.LinuxDevice) bool) []specs.LinuxDevice {
```

Comment:
I don't see any callers of this function, do we still need it?

Reply:
You're right; I'm not using it anymore, so will close this PR now.